syntax = "proto3";

package helloworld;

message HelloRequest {
  string name = 1;
}

message HelloReply {
  string message = 1;
}

message PDFRequest {
  string filename = 1;
  bytes file_content = 2;
}

message PDFReply {
  string message = 1;
}

service Greeter {
  rpc SayHello (HelloRequest) returns (HelloReply);
  rpc SendPDF (PDFRequest) returns (PDFReply);
}
